lib/commands/WeatherCommand.js

```diff
@@ -16,7 +16,11 @@ function getWeather(context, zip) {
     })
 
     weather.current({ method: 'zip', ZIPcode: zip, units: 'imperial' }, function(err, data) {
-        message.reply('the current weather for ' + data.name + ' is ' + data.weather[0].description + ', :thermometer: temperature is ' + data.main.temp + ' degrees. :droplet: humidity ' + data.main.humidity + '%, :wind_blowing_face: wind speed: ' + data.wind.speed + ' mph.')
+        if(!data.name) {
```

Comment:
`typeof data.name === 'undefined'` is a better check here, since `!data.name` will evaluate to a truthy expression when `data.name` is defined, but the empty string.